ansible/group_vars/all/system.yaml

```diff
@@ -13,7 +16,12 @@ docker_rpm_repository_url_upstream_mirror: "https://packages.d2iq.com/download.d
 docker_rpm_repository_url: "https://packages.d2iq.com/konvoy/stable/linux/centos/{{ ansible_distribution_major_version|int }}/x86_64"
 docker_rpm_gpg_key_url: "{{ docker_rpm_repository_url }}/gpg.asc"
 
+
+docker_rpm_suse_repository_url: "https://packages.d2iq.com/konvoy/stable/linux/opensuse/leap/{{ ansible_distribution_major_version|int }}/x86_64"
+docker_rpm_suse_gpg_key_url: "{{ docker_rpm_suse_repository_url }}/gpg.asc"
+
 # nvidia package
+nvidia_cuda_version: "460"
```

Comment:
Is the `cuda` word a synonym for `driver`?

Reply:
hmm its actually the framework... they sometimes mix it but just saw not in case of the driver packge... let me see if i can rename the vars

Reply:
nvidia is mixing the naming by themself. They call sometimes the driver package cuda-drivers. So I think its fair to stick with this name